### ISSUE TITLE
Use/UseDir ignore for platform specific scripts

### ISSUE THREAD

**Issue by Tricky1975** [jpbubble/Base#4]
Although Bubble is supposed to be "above the OSes", and thus to be as platform independent as possible, you never escape platform specific code. A scripting language like Lua is unfortunately not an exception.

These suffixes will only work on specific oses
- __win, __windows -> Windows only
- __darwin, __mac -> Mac only (or other darwin driven OSes)
- __lin, __linux -> Linux only

I will also put in the "not" keyword.

In other words "myscript__not_windows.lua" will be ignored on Windows, but not on Mac or in Linux.

These ignores are silent. You can just call the files normally on any platform, the Bubble engine will just ignore them when not applicable. No messages or errors will be thrown.

Be careful on using two underscores "__", as they may be used for other things in the future, leading to crazy effects :P
